src/XamlX.IL.Cecil/CecilReflectionImporterProvider.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Reflection;
+using Mono.Cecil;
+
+namespace XamlX.TypeSystem;
+
+internal class CecilReflectionImporterProvider : IReflectionImporterProvider
+{
+    public IReflectionImporter GetReflectionImporter(ModuleDefinition module)
+    {
+        return new ReflectionImporter(module);
+    }
+
+    public class ReflectionImporter : DefaultReflectionImporter
```

Comment:
Not necessarily for the fix, but I decided to add to be safer and avoid bugs caused by `cecil.ImportType(typeof(string))` and so

Reply:
There seems to be no other places where reflection import was used. At least building Avalonia assemblies.